db/migrate/20221102183102_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateComments < ActiveRecord::Migration[7.0]
+    def change
+      create_table :comments do |t|
+        t.references :author, index: true, foreign_key: { to_table: :users }
+        t.references :post, index: true, foreign_key: true
+        t.string :text
```

Comment:
- [ ] Do the same as requested in the above comment. The text column should have the `text` data type to allow users to write many characters while commenting on a post.